bitcoin/src/crypto/key.rs

```diff
@@ -58,15 +58,13 @@ impl PublicKey {
     pub fn pubkey_hash(&self) -> PubkeyHash { self.with_serialized(PubkeyHash::hash) }
 
     /// Returns bitcoin 160-bit hash of the public key for witness program
```

Comment:
While the error is obvious from the type we can still add Errors section as usual.

Reply:
Sure, are you imagining that in a perfect world all functions that return `Result` have an errors section?

Reply:
I think so. But it's minimal importance if the error is absolutely obvious.

Reply:
Perhaps once the dust has settled on the error changes crate wide we start doing this.